manager/src/main/java/io/gingersnapproject/search/opensearch/OpenSearchBackend.java

```diff
@@ -56,6 +57,32 @@ public Uni<String> put(String indexName, String documentId, String jsonString) {
             });
    }
 
+   @Override
+   public Uni<String> putAll(String indexName, Map<String, String> document) {
+      if (document.isEmpty()) {
+         return Uni.createFrom().nullItem();
+      }
+
+      Uni<?> mappingUni = ensureMapping(indexName);
+      return mappingUni.onItem()
+            .transformToUni(___ -> {
+               Request request = new Request("POST", "/_bulk");
+
+               request.addParameter("refresh", "wait_for");
+
+               StringBuilder body = new StringBuilder();
+               for (Map.Entry<String, String> entry : document.entrySet()) {
+                  body.append(Json.object("index", Json.object("_index", indexName, "_id", entry.getKey())));
+                  body.append("\n"); // using \n and not the system line separator, since the value will be used by the server VM
+                  body.append(entry.getValue());
```

Comment:
do we need to escape the line breaks in the value?

Reply:
good question! in this case it seems that we don't, trying to escape it I get an error from the OpenSearch server.

Reply:
a quick way to simulate this in the test is when creating the elements for the `documents` Map, use the `Json#toPrettyString` instead of `Json#toString`. But TBH not sure how to tackle this. If we only escape, it would turn the value into a string instead of an object.

Maybe we leave it as it is and just return the malformed error?

Reply:
The OpenSearch (and Elasticsearch) bulk indexing API does not expect a parsable JSON, I know that this is strage, but they are their api :) see https://opensearch.org/docs/1.2/opensearch/rest-api/document-apis/bulk/

Reply:
I think that the payload is tested here -> https://github.com/fax4ever/gingersnap-cache-manager/blob/2c8c8449a1d47bae2516b46a905b0051c0933deb/manager/src/test/java/io/gingersnapproject/search/SearchBackendTest.java#L45

Reply:
If I try to escape the value, the test fails.